### PR TITLE
new package bsdfetch

### PR DESCRIPTION
**Pull request by saltedcoffii**
- [x] x86_64
- [x] i686
- [x] armv7l/aarch64

### CODE REVIEW

**Comment by uberhacker** [comment]
Rather than replacing the package, why not update it with the bsdfetch?

**Comment by saltedcoffii** [comment]
OHHH. bsdfetch and go-fetch are two completely different packages that do two completely different things.

File: packages/go_fetch.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  description 'fetch makes it easy to download files, folders, or release assets from a specific commit, branch, or tag of a public or private GitHub repo.'
+  homepage 'https://github.com/gruntwork-io/fetch/'
+  @_ver = '0.4.2'
+  version @_ver
+  license 'LGPL-2.1'
+  compatibility 'all'
+  source_url 'https://github.com/gruntwork-io/fetch.git'
+  git_hashtag 'v' + @_ver
+
+  depends_on 'go'
```
Comment (uberhacker) [inline packages/go_fetch.rb:13]: Build only dependency?

File: packages/fetch.rb [hunk 2]
```diff
@@ -1,37 +1,32 @@
...
+              -DUSE_SYSTEM_SSL=ON .."
+    end
+    system 'samu -C builddir'
   end
 
   def self.install
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "go build -ldflags \"-X main.VERSION=v0.1.1\" -o #{CREW_DEST_PREFIX}/bin/fetch"
+    system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+    if CREW_LIB_SUFFIX != ''
```
Comment (uberhacker) [inline packages/fetch.rb:26]: Change to `if CREW_LIB_SUFFIX == '64'` for clarity and in case a new suffix is introduced?

Comment (saltedcoffii) [reply packages/fetch.rb:26]: If a new suffix is introduced, this logic will handle it. Although I can see your clarity point. Ping me if you'd still like this changed to `if CREW_LIB_SUFFIX == '64'`

Comment (uberhacker) [reply packages/fetch.rb:26]: It may or may not handle it correctly, though.  I don't think it's a good assumption.

Comment (saltedcoffii) [reply packages/fetch.rb]: okay! see 7c541b66

File: packages/go_fetch.rb (hunk 1 above)
Comment (saltedcoffii) [reply packages/go_fetch.rb]: Turns out it was, thank you

**Pull request merged**